### PR TITLE
Update example intro and simplified WNLI cards

### PR DESCRIPTION
**Pull request by yoavkatz**
since WNLI is used in the demos -  simplified it , so each processing effects one field, and it used explicit splitting.

### CODE REVIEW

File: docs/docs/examples.rst [hunk 1]
```diff
@@ -3,27 +3,31 @@
 Examples ✨
 ==============
 
-Here you find complete examples of using Unitxt
+Here you find complete examples showing how to perform different tasks using Unitxt. 
+Each example is self contained python file that you can run and later modify.
```
Comment (yoavkatz) [inline docs/docs/examples.rst:7]:
```suggestion
Each example is a self contained python file that you can run and later modify.
```

File: docs/docs/examples.rst [hunk 2]
```diff
@@ -3,27 +3,31 @@
...
+       | :ref:`Relation template in catalog <catalog.templates.classification.multi_class.relation.default>`.
+   * - Evaluate your question-answering dataset 
      - Demonstrates how to evaluate a user QA answering dataset in a standalone file using a user defined task and template.
      - `code <https://github.com/IBM/unitxt/blob/main/examples/standalone_qa_evaluation.py>`_
      - :ref:`Add new dataset tutorial <adding_dataset>`.
-   * - Evaluate a user defined open QA dataset - reusing existing catalog assets
-     - Demonstrates how to evaluate a user QA answering dataset using the predefined open qa task and templates.
+   * - Evaluate your question-answering dataset  - reusing existing catalog assets
+     - Demonstrates how to evaluate a user QA dataset using the predefined open qa task and templates.
        It also shows how to use preprocessing steps to align the raw input of the dataset with the predefined task field.
```
Comment (yoavkatz) [inline docs/docs/examples.rst:31]:
```suggestion
       It also shows how to use preprocessing steps to align the raw input of the dataset with the predefined task fields.
```

File: prepare/cards/wnli.py [hunk 3]
```diff
@@ -11,24 +12,18 @@
...
     loader=LoadHF(path="glue", name="wnli"),
     preprocess_steps=[
-        "splitters.small_no_test",
-        RenameFields(
-            field_to_field={
-                "sentence1": "text_a",
-                "sentence2": "text_b",
-            }
+        SplitRandomMix(
+            {"train": "train[95%]", "validation": "train[5%]", "test": "validation"}
```
Comment (eladven) [inline prepare/cards/wnli.py:16]: Isn't it the same as splitters.small_no_test? Why not using it?

**Comment by eladven** [comment]
Hi. I understand the one-field processing. However I thought that the best practice is to use predefined artifacts if exists

File: prepare/cards/wnli.py (hunk 3 above)
Comment (yoavkatz) [reply prepare/cards/wnli.py:16]: See this issue #925.    I think these catalog splitters are not clearly named  and hence not understood by users , especially in initial examples.

**Pull request merged**